src/main/java/it/unisa/Amigo/gruppo/services/GruppoService.java

```diff
@@ -1,29 +1,31 @@
 package it.unisa.Amigo.gruppo.services;
 
-import it.unisa.Amigo.gruppo.domain.ConsiglioDidattico;
-import it.unisa.Amigo.gruppo.domain.Dipartimento;
-import it.unisa.Amigo.gruppo.domain.Persona;
-import it.unisa.Amigo.gruppo.domain.Supergruppo;
+import it.unisa.Amigo.gruppo.domain.*;
 
 import java.util.List;
 
 /**
  * Questa interfaccia definisce i metodi  per la logica di Business del sottositema "Gruppo"
  */
 public interface GruppoService {
-    List<Persona> visualizzaListaMembriSupergruppo(int idSupergruppo);
-    List<Persona> visualizzaListaMembriConsiglioDidattico(int idConsiglioDidattico);
-    List<Persona> visualizzaListaMembriDipartimento(int idDipartimento);
-    List<Supergruppo> visualizzaSupergruppi(int idPersona);
-    List<ConsiglioDidattico> visualizzaConsigliDidattici(int idPersona);
-    List<Dipartimento> visualizzaDipartimenti(int idPersona);
-    ConsiglioDidattico findConsiglioBySupergruppo(int idSupergruppo);
+    List<Persona> findAllMembriInSupergruppo(int idSupergruppo);
+    List<Persona> findAllMembriInConsiglioDidattico(int idConsiglioDidattico);
+    List<Persona> findAllMembriInDipartimento(int idDipartimento);
     List<Persona> findAllMembriInConsiglioDidatticoNoSupergruppo(int idSupergruppo);
+    List<Persona> findAllMembriInGruppoNoCommissione(int idSupergruppo);
     Persona findPersona(int id);
-    Supergruppo findSupergruppo(int id);
+    Persona getAuthenticatedUser();
```

Comment:
Si potrebbe spostare in un altro service, magari in autenticazione, e restituire un oggetto di tipo user e da lì navigare la relazione e ottenere la persona. Per ora lasciamolo qua.